### ISSUE TITLE
DAPP_REMAPPINGS don't work in makefile

### ISSUE THREAD

**Issue by TransmissionsDev** [dapphub/dapptools#673]
With a `Makefile` like this:

```make
# Dapp Build Config.
export DAPP_REMAPPINGS=$(dapp remappings)

# Install, update, build and test everything.
all: solc install update build test
# Install proper solc version.
solc:; nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions.solc_0_8_6
# Install npm dependencies.
install:; npm install 
# Install dapp dependencies.
update:; dapp update

# Compiles the project.
build:; dapp build
# Test the project and only log verbose info for failed tests.
test:; dapp test --verbosity 1
# Test the project and log verbose info for everything.
vtest:; dapp test --verbosity 3
# Run a test using the HEVM interactive debugger.
debug:; dapp debug
```

Running `make build` shows these errors which indicate `DAPP_REMAPPINGS` is not working properly:

```sh

❯ make build
dapp build
+ dapp clean
+ rm -rf out
ParserError: Source "ds-test/test.sol" not found: File not found.
 --> src/tests/Vault.t.sol:4:1:
  |
4 | import "ds-test/test.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^

ParserError: Source "ds-test/test.sol" not found: File not found.
 --> src/tests/VaultFactory.t.sol:4:1:
  |
4 | import "ds-test/test.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^


make: *** [build] Error 1
```

This is odd because using this exact same env var in `.dapprc` works completely fine!

Here's a reproduction branch to toy around with locally (just clone and run `make`): 

https://github.com/Rari-Capital/vaults/tree/makefile-remappings-not-working

**Comment by d-xo** [comment]
I'm not sure that this had anything to do with the makefile. I think this isn't working because `dapp remappings` is failing with the same open zeppelin error from https://github.com/dapphub/dapptools/issues/674. 

I got the same behaviour when I put `DAPP_REMAPPINGS=$(dapp remappings)` into a `.dapprc`.

**Comment by jo-es** [comment]
Hi. I'm not sure if that issue has been resolved by #674.  I'm on dapp 0.33.0 on Mac OS and I experience the behaviour described by @TransmissionsDev. Running `DAPP_REMAPPINGS=$(dapp remappings) dapp build` directly via the command line works just fine. Setting `DAPP_REMAPPINGS`  in the Makefile is the only case where it's not working properly.

**Comment by MJarirUddin** [comment]
I got the same erorr and used this to remove it:
`import "../lib/ds-test/src/test.sol";`
works for me!
